charts/web-ui/templates/service.yaml

```diff
@@ -11,4 +11,4 @@ spec:
   selector:
     app: tensorleap-web-ui
   sessionAffinity: None
-  type: NodePort
+  type: ClusterIP
```

Comment:
Isnt this the exact change that caused issues with the last helm charts pr?
Also, did you check it works on onprem because I remember my onPrem telling me it doesnt support ClusterIp type

Reply:
AFAIK, the error we got is:
```
Upgrade "tensorleap" failed: cannot patch "tensorleap-minio" with kind Service: Service "tensorleap-minio" is invalid: spec.ports[0].nodePort: Forbidden: may not be used when `type` is 'ClusterIP'
```
(saw [this comment](https://tensorleap.slack.com/archives/CUB6LPDGA/p1684858374498959?thread_ts=1684857702.887979&cid=CUB6LPDGA) by @doronHarnoy )

Indeed, in this PR i've [removed the field](https://github.com/tensorleap/helm-charts/pull/110/files#diff-68ec96103cf33b3e5aa3005f353d5e40c0367d700fff370b1cc19f715e5c2d01L11) from `minio` service 

I've checked:
1. Upgrading installation with with `helm upgrade`
2. Manually installation from scratch using local helm

Right after merging I'll also verify:
1. Upgrade existing installation in the normal way
2. Installationg from scratch
These 2 things are pretty hard to test without merging